shuup/front/templates/shuup/front/macros/navigation.jinja

```diff
@@ -1,11 +1,12 @@
 {% extends "shuup/front/macros/theme/navigation.jinja" %}
+{% from "shuup/front/macros/general.jinja" import render_field with context %}
 
-{% macro render_navigation() %}
+{% macro render_navigation(quick_drop=True) %}
```

Comment:
I guess `quick_drop` name is not too intuitive. this should be a more explained parameter name, like `show_quick_login`